best-practices-metadata-and-extension.md

```diff
@@ -120,34 +122,46 @@ Each asset that points to measurements should provide the fields defined below.
   - `eo:common_name` (if applicable)
   - `eo:center_wavelength` and `eo:full_width_half_max`
 
-### SAR Measurements
+## SAR Measurements
 
-- **SAR Extension**  (v1.2 or a later non-breaking version)
-  - `sar:polarizations` (in the order as they should appear in the datacube)
+- **[SAR Extension](https://github.com/stac-extensions/sar)**  (v1.2 or a later non-breaking version)
+  - `sar:polarizations` (in the order they should appear in concatenated data structure)
 
-### Other Measurements
+- **[Product Extension](https://github.com/stac-extensions/product)**
+  - `product:type`
+   
+    If mixing multiple product types within a single Item, this can be used to specify per asset.
```

Comment:
Although it's allowed in assets, a product for me is a item and can't be an individual asset. If the product:type between assets differs, I think it should be a separate item.